<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuFrame</class>
 <widget class="QFrame" name="MenuFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="sound_pref" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="sound">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Sound</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="player_data" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Avatar</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nickname</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>140</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="playerListLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="adminWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>221</width>
     <height>361</height>
    </rect>
   </property>
   <widget class="QSpinBox" name="maxPlayersBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="maxPlayers_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Max players</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startGameButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start game</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="playersList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
   </widget>
   <widget class="QToolButton" name="mapLeftButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>26</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::LeftArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="mapRightButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>26</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="arrowType">
     <enum>Qt::RightArrow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Map</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="randomMapCheck">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QComboBox" name="mapOptionCBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="kickButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>93</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>kick</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="previewGraphicsView">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>281</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="serverStatus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="network_pref" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="isServer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>119</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Server IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>119</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Server port</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="serverIp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="serverPort">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="serverButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>119</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin password</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="password">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkDebugMode">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>180</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enables single player game, more bonus, no countdown, etc.</string>
    </property>
    <property name="text">
     <string>Debug Mode</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="IpStatHolder" native="true">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnectButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
